README.md

```diff
@@ -116,3 +116,22 @@ To opt out of tracking, please go to the raw `.ipynb` files and remove the follo
     "![Impressions](https://PixelServer20190423114238.azurewebsites.net/api/impressions/ComputerVision/classification/notebooks/21_deployment_on_azure_container_instances.png)"
 ```
 
+
+## Install this repository via PIP
+A [setup.py](setup.py) file is provided in order to simplify the
+installation of this utilities in this repo from the main directory.
+
+```
+pip install -e .
```

Comment:
if you do this, I'm not sure if it will add the tests, tools and notebooks to the library, I would try `pip install -e utils_cv`

Reply:
It would only add `utils_cv`.

Reply:
perfect